plugins/GSdx/GSCaptureDlg.cpp

```diff
@@ -64,7 +64,7 @@ void GSCaptureDlg::OnInit()
 
 	m_codecs.clear();
 
-	_bstr_t selected = theApp.GetConfig("CaptureVideoCodecDisplayName", "").c_str();
+	_bstr_t selected = CComBSTR(theApp.GetConfig("CaptureVideoCodecDisplayName", "").c_str()).Detach();
```

Comment:
I don't know how bstr are handled correctly. I read a bit about it and was concerned about memory leaks. Will read more but on the first glance this seemed to be the recommended conversion from std::string to _bstr_t.

(Sorry to say but string handling is really a mess in c++. Probably it is completely clear if you understand why these hundreds of string types exist but if you are used to unicode strings as default it is kind of complex.)